src/pages/academy/blog/posts/_style.js

```diff
@@ -405,10 +409,20 @@ export const PreviewContainer = styled(Box)`
     }
 `
 export const SocialComponentsWrapper = styled(Flex)`
+    display: flex;
+    align-items: center;
     justify-content: space-between;
-    border-top: 1px solid var(--color-grey-6);
-    padding-top: 24px;
-    margin: 40px 0;
+    padding-top: 60px;
+    margin: 8px 22px;
+    width: 0;
+
+    @media ${device.tabletL} {
+        display: flex;
+        justify-content: flex-end;
+        width: 0;
```

Comment:
Do we need this, seems like it has been set above?

Reply:
Thanks @suthesan-deriv  for all comments you did good job! :) 

For this issue we really need change justify-content, because in mobile version we have one button, not 4 icons